contrib/garyburd/redigo/redigo.go

```diff
@@ -19,10 +19,15 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/telemetry"
 
 	redis "github.com/garyburd/redigo/redis"
 )
 
+func init() {
+	telemetry.LoadIntegration("garyburd/redigo/redis")
```

Comment:
Should this just be `garyburd/redigo`?

Reply:
Oh strange. We named this package incorrectly based on our contrib requirements. So @lievan used the name of the package instead of the name of the directory.. which I would actually say is correct. Wdyt?

Reply:
Looks like this has happened in a few places (e.g. contrib/google.golang.org/api/api.go)

Reply:
Based on our convo, yep it should be `garyburd/redigo` to match the component tag 👍